### DISCUSSION TITLE
Sync taking a long time

### DISCUSSION THREAD

**Discussion by smth** [floccusaddon/floccus#1029] [Q&A] [answered]
I'm trying out Floccus and have a couple of questions about my recent sync that took a long time. First here's what I've done:

1. I set up the extension on one browser (using webDAV via pcloud) which had about 150 bookmarks in it
2. Synced in a few seconds 🎉
3. Switched to another browser with no bookmarks, and added the same account
4. Synced in a few seconds 🎉
5. Went back to the first browser, it started syncing
6. The sync took about 30 mins to complete 😕

So my questions are:

- Is it normal for syncing to take this long (perhaps once you have multiple browsers containing bookmarks)?
- Is it safe to close the browser while syncing?

**Comment by marcelklehr** [comment, marked as answer]
> took about 30 mins to complete

That sounds a lot like the lock timeout. When a sync process is interrupted it leaves behind a lock on the webDAV server, which tells other instances that a sync is in progress. Floccus will wait for half an hour for the lock to be released and continue anyway if it isn't, assuming that another sync process was interrupted.

**Reply by sevmonster** [reply]
I am using Nextcloud and see the same kind of issue.
